framework/CodeInterfaces/PHISICS/phisicsdata.py

```diff
@@ -0,0 +1,961 @@
+          keff = [line.split()[-1]]
+        if re.search(r'error for the eigenvalue', line) :
+            errorKeff = [line.split()[-1]]
+    return keff, errorKeff
+
+  def isNumber(self,line):
+    """
+      Checks if a string is an integer.
+      @ In, line, list, list of strings
+      @ Out, isNumber, Boolean, True if integer, False otherwise
+    """
+    if line != []:
+      try:
+        int(line[0])
+        return True
+      except ValueError:
+        return False
+        
+  def isFloatNumber(self,line):
+    """
+      Checks if a string is a float number.
+      @ In, line, list, list of strings
+      @ Out, isFloatNumber, Boolean, True if float, False otherwise
+    """
+    if line != []:
+      try:
+        float(line[0])
+        return True
+      except ValueError:
+        return False
+   
+  def convertInDays(self,instantTimeSteps):
+    """
+      Converts the INSTANT time steps (seconds) into days.
+      @ In, instantTimeSteps, list
+      @ Out, timeSteps, list
+    """
+    timeSteps = []
+    for i in xrange (0, len(instantTimeSteps)):
+      timeSteps.append('%.6E' % (float(instantTimeSteps[i]) / (24 * 60 * 60)))
+    return timeSteps
+   
+  def declareDict(self,typeOfParameters,numbering):
+    """
+      Declares the reaction rate dictionary.
+      @ In, typeOfparameter, string, either 'reaction rate' or 'FissionMatrix', to select which matrix is to be parsed
+      @ In, numbering, dict, Column number (key) parameter number (value)
+      @ Out, dict, dictionary, dictionary with reaction rate keys, and empty values
+    """
+    dict = {}
+    if self.printSpatialRR is True:
+      for i in xrange (1,int(self.Ngroups) + 1):
+        dict[str(i)] = {}
+        for j in xrange (1,int(self.Nregions) + 1):
+          dict[str(i)][str(j)] = {}
+          for k in xrange (0, len(numbering)):
+            if numbering.keys()[k] == 'Group':
+              pass
+            else:
+              dict[str(i)][str(j)][numbering.keys()[k]] = {}
+    if self.printSpatialRR is False:
+      for k in xrange (0, len(numbering)):
+        if numbering.keys()[k] == 'Group':
+          pass
+        else:
+          dict[numbering.keys()[k]] = {}
+    if typeOfParameters == 'reactionRates':   # Puts the reaction rate labels in a list
+      self.paramList = []
+      for key in numbering.iterkeys():
+        self.paramList.append(key)
+        
+    if typeOfParameters == 'FissionMatrix':   # Puts the fission matrix labels in a list
+      self.matrixList = []
+      for key in numbering.iterkeys():
+        self.matrixList.append(key)
+    return dict
+        
+  def getReactionRates(self,numberOfMPI):
+    """
+      Gets the reactions rates, power for each group in PHISICS
+      @ In, None
+      @ Out, ReactionRateDict, dictionary, dictionary containing the reaction rate info
+    """
+    flagStart = 0
+    count = 0
+    countTimeStep = 0
+    for mpi in xrange (0,numberOfMPI): # parse all the segmented files
```

Comment:
rework this method

Reply:
I cleaned up thre 'regular' comments, how would you turn it into?